composer.json

```diff
@@ -80,25 +80,35 @@
         "sort-packages": true
     },
     "scripts": {
+        "post-autoload-dump": [
+            "@install-tools"
+        ],
         "cs:check": "@php php-cs-fixer fix --dry-run --diff",
         "cs:fix": "@php php-cs-fixer fix",
-        "dev-tools:check": [
-            "./dev-tools/check_file_permissions.sh",
-            "./dev-tools/check_trailing_spaces.sh"
-        ],
-        "dev-tools:docs": "@php dev-tools/doc.php",
-        "dev-tools:install": "@composer -d dev-tools install",
-        "docs": "@dev-tools:docs",
+        "docs": "@php dev-tools/doc.php",
+        "install-tools": "@composer -d dev-tools install",
         "mess-detector": "@php dev-tools/vendor/bin/phpmd . ansi dev-tools/mess-detector/phpmd.xml --exclude vendor/*,dev-tools/vendor/*,dev-tools/phpstan/*,tests/Fixtures/*",
-        "phpstan": "@php -d memory_limit=256M dev-tools/vendor/bin/phpstan analyse",
-        "phpstan:baseline": "@php -d memory_limit=256M dev-tools/vendor/bin/phpstan analyse --generate-baseline=./dev-tools/phpstan/baseline.php",
+        "normalize": [
+            "@composer normalize -d dev-tools --dry-run ../composer.json",
+            "@composer normalize -d dev-tools --dry-run composer.json"
```

Comment:
```suggestion
            "@composer normalize --working-dir=dev-tools --dry-run ../composer.json",
            "@composer normalize --working-dir=dev-tools --dry-run composer.json"
```

Reply:
But why? What value does it provide, besides more text printed when running script?

Reply:
Looking at `grep` discussion above, there was `-P`, not `--perl-regexp` 😉. I don't think there's any convention for that in the project.

Reply:
> But why? What value does it provide, besides more text printed when running script?

It is more readable, a person who reads it, knows from the name what it does, no need to know what `-d` stands for.

Reply:
> I don't think there's any convention for that in the project.

So let's make a one - descriptive option over the short ones that require knowing them.

In this very PR there is `@install-tools --quiet`, but not `@install-tools -q`.

Reply:
👍🏻 for descriptive variants, there is always some command and some one-letter toggle that JohnDoe is not familiar with, if with no-costs we can avoid asking ppl to google for it, I like it

Reply:
I let myself to commit the changes and then merge the PR. we can always improve it further in next PR